### ISSUE TITLE
remove coding schemes like S00L02_... from  from html files

### ISSUE THREAD

**Issue by rmonajemi** [LUMC/rcourse#6]
With a schedule change the hard links in the documents will break.

**Comment by SzMK** [comment]
This is not a good way to go.
Having SxxLyy organises the files so students can see order of lectures.
Linking between files can be done in another way.

**Comment by rmonajemi** [comment]
> This is not a good way to go.
> Having SxxLyy organises the files so students can see order of lectures.
> Linking between files can be done in another way.

I do not agree. Students can see the order on the Schedule page and not through file ids. The issue of the order can be solved through config file programmatically.
